core/src/main/java/org/apache/calcite/plan/RelOptMaterializations.java

```diff
@@ -50,6 +50,12 @@
  */
 public abstract class RelOptMaterializations {
 
+  @Deprecated // to be removed before 2.0
+  public static List<Pair<RelNode, List<RelOptMaterialization>>> useMaterializedViews(
+      final RelNode rel, List<RelOptMaterialization> materializations) {
+    return useMaterializedViews(rel, materializations, null);
```

Comment:
Should the third parameter be an empty list rather than null (I think otherwise we may hit an NPE)?

Reply:
Thanks, update the code.